src/lib/storage/fake_device_factory.rb

```diff
@@ -242,7 +242,7 @@ def create_partition_table(parent, args)
       # @return [Fixnum]
       def str_to_ptable_type(string)
         # Allow different spelling
-        string = "msdos" if string.downcase == "ms-dos"
+        string = "msdos" if string.casecmp("ms-dos").zero?
```

Comment:
I agree with @ancor, writing a helper for String to make it readable again is actually against the original reason why Rubocop complains... In that case I'd rather keep the current style and disable the Rubocop check.

But there is one important thing: if you cannot remember the exact syntax for the new style comparison you can still use the old way (`foo.downcase == "bar"`) and let `rubocop -a` fix it for you.

(BTW the case insensitive comparison is not used much in YaST so we are actually solving a minor case...)

So to move forward I'm voting for the new syntax :wink: 

Let's use the "reaction" button to vote here:
-  :+1: for the new style
-  :-1: for the old style